### PR TITLE
pay route: render fallback description if no amount is specified

### PR DESCRIPTION
**Pull request by Aierie**
# With amount
Preview shown on twitter for a link shared by 'Kiel's NFT Market' to request payment, with an amount

# Without amount
Preview shown on twitter for a link shared by 'Kiel's NFT Market' to request payment, with no amount.

# Without amount, missing merchant customization
Preview shown on twitter for a link shared by an unknown business to request payment, with no amount.

### CODE REVIEW

**Review by jurgenwerk — COMMENTED** [review]
Can we add a generic description in this case? It seems a bit odd that we include the tag conditionally

Maybe something in the lines of "Use Cardstack wallet to pay ${name}"?

File: packages/ssr-web/app/templates/pay.hbs [hunk 1]
```diff
@@ -1,23 +1,29 @@
...
-  (concat (or this.merchantInfo.name @model.merchantSafe.address) ' requests payment')
-  (concat 'Pay ' this.cleanedAmounts.displayed.amount)
-  as |title description|
+  (or this.merchantInfo.name @model.merchantSafe.address)
+  as |business|
 }}
-  {{page-title title replace=true}}
+  {{#let
+    (concat business ' requests payment')
+    (concat 'Use Card Wallet to pay ' (or this.cleanedAmounts.displayed.amount business))
```
Comment (backspace) [inline packages/ssr-web/app/templates/pay.hbs:7]: Maybe we should get input on this? I had “Card Wallet” in a previous iteration and was [asked](https://discord.com/channels/584043165066199050/813819872177815572/945823567864545282) to remove it 😆

**Review by backspace — APPROVED** [review]
I am in favour of this but I think we should confirm wording 🤓

File: packages/ssr-web/app/templates/pay.hbs (hunk 1 above)
Comment (lukemelia) [reply packages/ssr-web/app/templates/pay.hbs:7]: @Aierie please add a screenshot to this PR and ask Kiel to provide input on the copy.

**Comment by Aierie** [comment]
I'm having a strange issue where the qr code test selector won't show up locally, even though CI passes. It seems like there is a cached build of Boxel on my machine that is interfering with the test. Will continue this on Monday.

**Comment by backspace** [comment]
`yarn test` succeeded for me locally 🎉

image

**Pull request merged**